py/server/tests/testbase.py

```diff
@@ -15,7 +15,6 @@
 
 _JTableTools = jpy.get_type("io.deephaven.engine.util.TableTools")
 
-
```

Comment:
Isn't 2 spaces some kind of Python convention?

Reply:
yes